dlrover/python/master/node/dist_job_manager.py

```diff
@@ -1066,14 +1066,15 @@ def update_node_resource_usage(
             return
         node.update_resource_usage(cpu, memory, gpu_stats)
         cpu_percent = node.used_resource.cpu / node.config_resource.cpu
-        if cpu_percent < _dlrover_context.hang_cpu_usage_percentage:
-            if node.start_hang_time == 0:
-                now = datetime.now()
-                node.start_hang_time = now.timestamp()
-        else:
-            if node.start_hang_time > 0:
-                now = datetime.now()
-            node.start_hang_time = 0
+        if not self.is_all_reduce_type_job():
```

Comment:
The torch training does not necessarily have to run on the GPU; it can also run on the CPU.